src/clr-addons/components.clr-addons.scss

```diff
@@ -367,4 +380,25 @@ input:not([type='time']) {
   .content-panel-content-wrapper {
     padding: 0.5rem;
   }
+
+  //prevents the upper body part from being hidden by the fixed header
+  body {
+    padding-top: $clr-header-height;
+  }
+
+  //main-container needs the display:block property in order to work properly on safari
+  .main-container {
+    display: block;
```

Comment:
with those changes on the layout, u-main-container stops working correctly.
plus it's a heavy change of the standard clarity layout which I don't really want to do!
if there are any major issues on specific browsers we need to fix it in clarity itself.
just tested systemmanagement in safari and iphone 7 and it works perfectly fine with current implementation on DEV

Reply:
As stated in a VU3MY userstory, the title-bar needs to be scrollable, whereas header and footer need to be sticky. Due to the current page layout, this can only be achieved by removing the display:flex property and fixing header and footer.

Reply:
hmm ok, can we then please create a separate file called mobile.scss and put all the stuff inside there to find it easier.
additionally the media selector only has to be defined at the first line